8.8/vips.modules

```diff
@@ -582,8 +587,9 @@
     autogenargs="--without-x --without-lzma --without-modules --without-openexr --without-heic --without-gvc --without-lqr --without-magick-plus-plus --disable-largefile --without-rsvg --without-zlib --disable-openmp">
     <branch
       repo="magick"
-      module="ImageMagick-6.9.10-42.tar.xz"
-      version="6.9.10-42"/>
+      module="6.9.10-43.tar.gz"
+      version="6.9.10-43"
+      checkoutdir="ImageMagick6-6.9.10-43"/>
```

Comment:
-42 has already been deleted :-( perhaps downloading from the github repo will be safer.

Reply:
Indeed downloading from the GitHub repo will be safer. I didn't try it because I'm not sure if they're doing any post-processing on the tarballs.

Reply:
I assumed since they'd set up releases it was OK, and it seems to build. I'll see if I can see any differences.